packages/react-select/src/Select.tsx

```diff
@@ -77,6 +77,8 @@ export interface Props<
   'aria-invalid'?: AriaAttributes['aria-invalid'];
   /** Aria label (for assistive tech) */
   'aria-label'?: AriaAttributes['aria-label'];
+  /** Aria autocomplete */
+  'aria-autocomplete'?: 'none' | 'inline' | 'list' | 'both';
```

Comment:
inline is not a valid option for this component - https://www.w3.org/WAI/ARIA/apg/patterns/combobox/#wai-aria-roles-states-and-properties-6

Reply:
Hi @kethomas, Thanks for the review. I've removed the "inline" option. Please check.